### PR TITLE
Merge master into develop

### PR DESCRIPTION
**Pull request by omichel**
It looks something went wrong with the master into develop performed yesterday.
This PR aims at fixing this.

### CODE REVIEW

File: projects/samples/contests/robocup/protos/RobocupGoal.proto [hunk 1]
```diff
@@ -32,8 +32,13 @@ PROTO RobocupGoal [
   post_radius = 0.05 -- Diameter/width/depth must be between 0.08-0.12
   post_spacing = D + 2 * post_radius
   post_height = GH + post_radius
+<<<<<<< HEAD
```
Comment (stefaniapedrazzi) [inline projects/samples/contests/robocup/protos/RobocupGoal.proto:35]: All these lines are wrong and should not be committed.
These are the files used by the merge tool.

Comment (omichel) [reply projects/samples/contests/robocup/protos/RobocupGoal.proto:35]: Yes, I know, this a draft PR which I am fixing now.

**Comment by omichel** [comment]
I had to manually fix all the files in this PR. See also https://github.com/cyberbotics/webots/pull/2783#issuecomment-796540616

**Comment by omichel** [comment]
I believe I did a mistake yesterday when merging #2838 as I clicked "squash and merge"...

**Comment by stefaniapedrazzi** [comment]
:eyes:

**Review by stefaniapedrazzi — APPROVED** [review]
Thank you.

**Comment by omichel** [comment]
I will re-enable master/develop branch protection in the settings (and we will have to disable them each time we merge master into develop).

**Pull request merged**